elections/uk/settings.py

```diff
@@ -14,3 +14,62 @@
 TEMPLATE_CONTEXT_PROCESSORS = (
     "uk_results.context_processors.show_results_feature",
 )
+
+PEOPLE_LIABLE_TO_VANDALISM = {
+    2811, # Theresa May
+    1120, # Jeremy Corbyn
+    4546, # Boris Johnson
+
+    # Below we include the person ID of anyone who is currently a minister.
+    # This list was generated from:
+    #
+    #   https://github.com/mysociety/parlparse/blob/master/members/ministers-2010.json
+    #
+    # ... with this snippet of Python:
+    #
+    # import datetime, json
+    # with open(ministers-2010.json') as f:
+    #     ministers = json.load(f)
+    # for m in ministers['memberships']:
+    #     if not re.search(r'^(Minister|The Secretary of State|Deputy Prime Minister|The Prime Minister)', m.get('role', '')):
+    #         continue
+    #     today = str(date.today())
+    #     if not (today >= m['start_date'] and today <= m.get('end_date', '9999-12-31')):
+    #         continue
+    #     i = Identifier.objects.filter(identifier=m['person_id']).first()
+    #     if i is None:
+    #         continue
+    #     person = i.content_object
+    #     print '    {0}, # {1} ({2})'.format(person.id, person.name, m['role'])
+
+    2885, # David Davis (The Secretary of State for Exiting the European Union)
+    212, # Alan Duncan (Minister of State)
+    2832, # Michael Fallon (The Secretary of State for Defence)
+    451, # Liam Fox (The Secretary of State for International Trade and President of the Board of Trade)
+    918, # Nick Gibb (Minister of State (Department for Education))
+    3486, # Damian Green (The Secretary of State for Work and Pensions)
+    3445, # John Hayes (Minister of State (Department for Transport))
+    2811, # Theresa May (The Prime Minister)
```

Comment:
The output is generated by that Python snippet above and pasted in, so I think it's more useful to have the literal output of it included for easy comparison with subsequent runs (or trying different variants of that snippet). Since it's a set, as you note, it makes no difference to the setting and I think the usefulness of having the literal output outweighs the possible confusion from there being duplicates in the file.